prometheus_blackbox_exporter/templates/blackbox.yml.j2

```
@@ -0,0 +1,5 @@
+# {{ ansible_managed }}
+---
+{% if blackbox_exporter__modules %}
+{{ dict(modules=blackbox_exporter__modules) | to_yaml(sort_keys=false) }}
+{% endif %}
```

I think it'd be better to leave an empty dict than an empty config here.. Also, I recommend using `to_nice_yaml`
```suggestion
{{ dict(modules=blackbox_exporter__modules) | to_nice_yaml(sort_keys=false) }}
```

---

Ah, there are no defaults, okay. :+1: